app/controllers/works_controller.rb

```diff
@@ -451,8 +452,9 @@ def destroy
     begin
       was_draft = !@work.posted?
       title = @work.title
+      unescaped_title = CGI.unescapeHTML(title)
       @work.destroy
-      flash[:notice] = ts('Your work %{title} was deleted.', title: title)
+      flash[:notice] = ts("Your work %{title} was deleted.", title: unescaped_title)
```

Comment:
The good news is this is totally the right way to do this!

However, we're pretty deep into doing in the wrong way and just calling `.html_safe` on the `flash[:notice]`. After some brief discussion, we decided we'd prefer to be consistently wrong and use `.html_safe` until we can change the underlying issue with how we're saving these special characters in the database.